nexus/lib/structure.py

```diff
@@ -4989,7 +4989,13 @@ def get_number_of_atoms(self):
     def get_atomic_numbers(self):
         an = []
         for e in self.elem:
-            an.append(ptable[e].atomic_number)
+            if e[1:].isdigit(): # Index concatenated to one-character atomic symbol (e.g., C1)
```

Comment:
You should use the `is_element` function from `periodic_table.py`.

Something like:
```
iselem,e = is_element(e,symbol=True)
if not iselem:
    # some kind of error message
else:
    an.append(ptable[e].atomic_number)
#end if
```